### DISCUSSION TITLE
Installing FFMPEG Questions

### DISCUSSION THREAD

**Discussion by Kurosu22** [shish/shimmie2#1102] [Q&A] [answered]
Hi,
Sorry if this is a newbie question but I'm running Shimmie2 on a Synology NAS and was wondering how to install FFMPEG so I can get video thumbnails. Do I have to add FFMPEG separately or do I put it in the Synology files?

Thanks in advance and great job Shish and the team!

**Comment by shish** [comment]
I'm afraid I don't know anything about how synology works or runs third-party software ^^

If you're using a docker image, then that should come with all the dependencies (eg ffmpeg) pre-installed inside the container. If you're using git or zip files, then ffmpeg needs to be installed on the same system, but I've no idea how one would even start with that :S

**Reply by Kurosu22** [reply, marked as answer]
Thanks for the quick response!
I'm learning how to use docker images now and it's working great so far!
Thanks again!
